input/blog/2018-01-17-cake-v0.25.0-released.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Cake v0.25.0 released
+category: Release Notes
+author: devlead
+---
+
+Version 0.25.0 of Cake has been released.
+
+The first release of 2018 is here! It contains mostly improvements and bug fixes, but it also has one potential breaking change. 
+
+The most notable change in this release is that we no longer use the NuGet.exe CLI tool when installing addins, tools, modules and scripts form NuGet sources in our [preprocessor directives](/docs/fundamentals/preprocessor-directives).
+No we instead default to the in-process NuGet client introduced in Cake [v0.22.0](/blog/2017/09/cake-v0.22.0-released). This will not only fix many previous issues people seen with Cake's preprocessor NuGet handling, but also enable us to simplify boostrappers and pave the way to properly solve addin dependencies. It will also make make pure .NET Core environments work out of the box with `Cake.CoreCLR`.
+
+Also notable is that this release will likely be the last one before we migrate `Cake.CoreCLR` to .NET Core 2.x, we know this is much awaited! The migration work has been going on for some time now and most blockers have been sorted. So our preliminary plan is to have this migration done by the next major release.
+
+The new NuGet client shouldn't break any existing scripts, addins or tools, but you might need to clear your tools/addins folder as it has a slightly different folder structure. You can opt-out if you're having issues, by using Cake [configuration](https://cakebuild.net/docs/fundamentals/default-configuration-values) environment variable, file or argument.
```

Comment:
Wrap "tools/addins" in backticks so it shows as code, perhaps?

Reply:
sure.